_data/chains/eip155-8082.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Shardeum Sphinx 1.X",
+  "chain": "Shardeum",
+  "icon": "shardeum",
+  "rpc": ["https://sphinx.shardeum.org/"],
+  "faucets": ["https://faucet-sphinx.shardeum.org/"],
+  "nativeCurrency": {
+    "name": "Shardeum SHM",
+    "symbol": "SHM",
+    "decimals": 18
+  },
+  "infoURL": "https://docs.shardeum.org/",
+  "shortName": "Sphinx10",
+  "chainId": 8082,
+  "networkId": 8082,
+  "explorers": [
+    {
+      "name": "Shardeum Scan",
+      "url": "https://explorer-sphinx.shardeum.org",
+      "standard": "EIP3091"
```

Comment:
```suggestion
      "standard": "none"
```

Reply:
I will look into this soon. 

Thank you.

Reply:
Shardeum EIP-3091 Tests:

:green_circle: Pass: (3/4)

Transactions
<BLOCK_EXPORER_URL>/tx/<TX_HASH>

https://explorer-sphinx.shardeum.org/tx/0xcbd6626319f615d250794904eaa3930855f0805816e08ed40155b03ec8c26008

Accounts
<BLOCK_EXPORER_URL>/address/<ACCOUNT_ADDRESS>

https://explorer-sphinx.shardeum.org/address/0xc1202e7d42655f23097476f6d48006fe56d38d4f

ERC-20 Tokens
<BLOCK_EXPORER_URL>/token/<TOKEN_ADDRESS>

https://explorer-sphinx.shardeum.org/token/0xdbaa7dfbd9125b7a43457d979b1f8a1bd8687f37

:red_circle:  Fail: (1/4)

Shardeum supports cycles instead of blocks (Shardeum bundles) at the moment in the explorer: 

Blocks
<BLOCK_EXPORER_URL>/block/<BLOCK_HASH_OR_HEIGHT>

It is instead:

Cycles
<BLOCK_EXPORER_URL>/cycle/<CYCLE_HASH_OR_HEIGHT>

https://explorer-sphinx.shardeum.org/cycle/510

Therefore, Shardeum currently does not meet the full EIP-3091 path specification:
https://eips.ethereum.org/EIPS/eip-3091

We can set: 
```
"standard": "none"
```

Reply:
did u use some tool for the test?

Reply:
I did this manually with the URL examples I have.